src/Service/NotificationService.php

```diff
@@ -103,8 +100,13 @@ public function sendUserBookingNotification(
         $dateStart = $userBooking->start;
         $dateEnd = $userBooking->end;
 
-        $dateStart->setTimezone(new PhpDateTimeZone($this->bindNotificationTimezone));
-        $dateEnd->setTimezone(new PhpDateTimeZone($this->bindNotificationTimezone));
+        if (!empty($this->bindNotificationTimezone)) {
+            $dateStart->setTimezone(new PhpDateTimeZone($this->bindNotificationTimezone));
+            $dateEnd->setTimezone(new PhpDateTimeZone($this->bindNotificationTimezone));
+        } else {
+            // Handle the case where bindNotificationTimezone is empty
+            throw new \InvalidArgumentException('bindNotificationTimezone cannot be empty');
+        }
```

Comment:
Consider moving the check to the constructor and do it only once, i.e. add 

```php
if (!$this->bindNotificationTimezone) {
    throw new \InvalidArgumentException('bindNotificationTimezone cannot be empty');
}
```

in `__construct` and remove the (duplicated) checks elsewhere.